catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -0,0 +1,125 @@
+package ddf.catalog.transformer.input.pdf;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Calendar;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDDocumentInformation;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.util.ImageIOUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.InputTransformer;
+import ddf.catalog.transform.pdf.PdfThumbnailGenerator;
+
+public class PdfInputTransformer implements InputTransformer, PdfThumbnailGenerator {
+
+    public static final String PDF_CONTENT_TYPE = "pdf";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PdfInputTransformer.class);
+
+    private static final int RESOLUTION_DPI = 24;
+
+    private static final String FORMAT_NAME = "jpg";
+
+    private static final float IMAGE_QUALITY = 1.0f;
+
+    public PdfInputTransformer() {
+    }
+
+    @Override
+    public Metacard transform(InputStream input) throws IOException, CatalogTransformerException {
+        return transform(input, null);
+    }
+
+    @Override
+    public Metacard transform(InputStream input, String id)
+            throws IOException, CatalogTransformerException {
+        PDDocument pdfDocument = PDDocument.load(input);
+        MetacardImpl metacard = new MetacardImpl();
+
+        metacard.setId(id);
+        metacard.setMetadata("");
+
+        if (pdfDocument.isEncrypted()) {
+            LOGGER.debug("Cannot transform encrypted pdf");
+            // TODO (RCZ) - do we want to fail for any reason?
+            return metacard;
```

Comment:
That is probably reasonable.  I believe the metacard groomer adds some additional attributes for discoverability.